sms.go

```diff
@@ -44,6 +45,16 @@ func (sms *SmsResponse) DateSentAsTime() (time.Time, error) {
 	return time.Parse(time.RFC1123Z, sms.DateSent)
 }
 
+func whatsapp(phone string) string {
+	return fmt.Sprintf("whatsapp:%s", phone)
```

Comment:
Because this is such a simple string I would recommend changing this to:

return "whatsapp:" + phone

It's a little smaller, faster and just as readable.

Reply:
Good catch, just realized I pulled in "fmt" just for that. Thanks.